release-notes/v5.8.0.md

```diff
@@ -131,3 +131,7 @@
 #### <sub><sup><a name="4895" href="#4895">:link:</a></sup></sub> fix
 
 * @kirillbilchenko fixed a [bug](https://github.com/concourse/concourse/issues/3856) where the `concourse_workers_registered` metric would never go below 1, even when workers were pruned. #4895
+
+#### <sub><sup><a name="4947" href="#4947">:link:</a></sup></sub> fix
```

Comment:
am I doing this right?

In [./CONTRIBUTING.md](./CONTRIBUTING.md) it says that we should edit `latest.md`, but there's no such file (I guess that's because we're about to cut 5.8.0?)

thanks!

Reply:
I think `latest.md` should be created only if we're very close to cutting and this change can wait until the next release. But it looks like 5.8 was pending some documentation that hasn't been completed, so it's safe to sneak this one in 😄 

Here's the issue tracking what's going into 5.8, could you add this to the "review release notes" subheading: https://github.com/concourse/concourse/issues/4859